conda-forge.yml

```diff
@@ -4,9 +4,10 @@ bot:
   - 4.0.x
   - 5.0.x
   - 6.0.x
+provider:
```

Comment:
Is this required to test CUDA? Otherwise, I would prefer to keep them in cross-compilation mode.

Reply:
Unfortunately cross-compiling with CUDA didn't work well in my attempts above. So gave up. Think that will require more work in conda-forge generally.